tests/Microsoft.Identity.Test.Unit.net45/AuthorityAliasesTests.cs

```diff
@@ -39,14 +39,15 @@ public async Task AuthorityMigrationTestAsync()
 
             using (var httpManager = new MockHttpManager())
             {
-                httpManager.AddInstanceDiscoveryMockHandler();
 
                 var authorityUri = new Uri(
                     string.Format(
                         CultureInfo.InvariantCulture,
                         "https://{0}/common",
                         MsalTestConstants.ProductionNotPrefEnvironmentAlias));
 
+                httpManager.AddInstanceDiscoveryMockHandler(authorityUri.AbsoluteUri);
```

Comment:
this is showing up weird in github.  is there a spacing/indenting issue here?

Reply:
Wierd, but hopefully fixed, I auto-formatted the doc.